A38-admin-interface-api.md

```diff
@@ -166,7 +149,10 @@ server = ServerBuilder.forPort(50051)
 server.awaitTermination();
 ```
 
-Option 4: Create admin server
+The drawback for this option is dependency management. To date, the gRPC official admin services Channelz/Reflection/Health are distributed as separated package to keep the main package focus as a network library. If we apply this option, then we will void the separate effort.
```

Comment:
Couldn't we use the same registration pattern with this as we're going to use anyway?  This doesn't seem like the right reason not to go with this design.  (I'm not arguing that we should have chosen this design; I'm just saying that I don't think this paragraph accurately describes the reasons why we didn't choose it.)

Also, reflection and health are not relevant to this design.